### DISCUSSION TITLE
Unable to find project members from GET Project Member API

### DISCUSSION THREAD

**Discussion by jaydenlzx** [iTwin/itwinjs-core#4084] [Q&A] [answered]
Hello All,

I am trying to retrieve a particular user by using GET Project Member API. However, I could not locate him from the API results by using CTRL F to find his member ID. There isnt a next page to the list. 

![image](https://user-images.githubusercontent.com/80751784/184590709-eca42f2f-0b16-4deb-b2d1-d2a3720a0b49.png)

![image](https://user-images.githubusercontent.com/80751784/184590803-3b066037-f9f2-494d-98e2-f58748b7b6c9.png)

I tried to add him to the project environment using POST Add Project Member API. However, it shows that the member is already existing in the project environment.

![image](https://user-images.githubusercontent.com/80751784/184590431-73f679b1-4868-4388-b595-fc09aaa18338.png)

Has anyone encountered this issue before? Can anyone give me some pointers for me on how to resolve this issue?

Thank you.

**Comment by pmconne** [comment]
@davidhjones

**Comment by davidhjones** [comment, marked as answer]
Hello @jaydenlzx -- it seems that we have an issue with our paging that needs to be addressed. In short, your Project has a bunch of members which are no longer working at your company. This introduces a bug in the paging where it seems that there aren't anymore pages, but in fact there are more pages. 

As a workaround you can retrieve subsequent pages to see the additional team member:
`GET https://api.bentley.com/projects/69c70697-3747-4120-b185-dbd7d54388a0/members?$skip=100&$top=100`

In the meantime we will look into addressing the issue by fixing the paging bug to return users which are no longer working at your company -- this will give you the ability to cleanup these users by removing them from the team member list.

**Reply by jaydenlzx** [reply]
Hi David, thank you for your responses. 

How do you define that the members are no longer working? My colleagues' accounts may be disabled but it doesn't mean that they are not working on the project. 

How do i prevent such bugs from happening? 

Please do let me know if the bug is resolved and i will proceed to cleanup if there are any members that need to be removed from the project. 

Thank you.

**Reply by davidhjones** [reply]
Hey @jaydenlzx -- we are working to resolve this bug from happening, thus no cleanup will be needed on your side. In the meantime continue to fetch subsequent pages. Once we resolve this bug the paging issue will be resolved and there is no longer be "ghost" users on your Projects.

I'm hopeful this will be a quick implementation / fix, and I'll update you as we deploy a fix to production. Thanks for your patience.

**Reply by maksim-bnt** [reply]
Hi @jaydenlzx we applied the fix to the new access-control endpoint. https://developer.bentley.com/apis/access-control/operations/get-iTwin-members/ . Documentation explains new behavior. We also have weekly job that will clean up users that were flagged as missing from the Identity Management System. Please let us know if you have other questions.

**Reply by calebmshafer** [reply]
marking this as the answer as we have deployed the fix. @jaydenlzx if the issue has persisted please let us know. thanks!

**Comment by davidhjones** [comment]
Hi @jaydenlzx — please let us know if have any additional questions, or if everything has been address. If so, could you please close out this request, thanks!
